poetry/core/masonry/builders/builder.py

```diff
@@ -127,6 +141,15 @@ def find_files_to_add(self, exclude_build=True):  # type: (bool) -> list
                     continue
 
                 if file.is_dir():
+                    if self.format == "sdist" or self.format in include.formats:
+                        for f in Path(file).glob("**/*"):
```

Comment:
```suggestion
                        for f in file.glob("**/*"):
```

`file` is already a `Path` object here.